app/Services/Spotify/OAuth/SpotifyTokenManager.php

```
@@ -0,0 +1,181 @@
+     *
+     * @param string      $accessToken  The access token
+     * @param string|null $refreshToken The refresh token (optional, only provided on first auth)
+     * @param int         $expiresIn    Seconds until token expires
+     *
+     * @return void
+     */
+    public function storeToken(string $accessToken, ?string $refreshToken, int $expiresIn): void
+    {
+        $expiresAt = now()->addSeconds($expiresIn)->timestamp;
+
+        Cache::put(self::ACCESS_TOKEN_KEY, $accessToken, $expiresIn);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'storeToken'. (undefined)

(StaticAccess)